### ISSUE TITLE
Feature Request: Pin tool to organise wires

### ISSUE THREAD

**Issue by OliverEGreen** [DynamoDS/Dynamo#8027]
## Dynamo version
Whenever possible

## Operating system
Windows 7

## Feature

@Racel 

It would be great if Dynamo had a native 'pin' tool to help us lock wires in place. This would help create more legible graphs and avoid the problem of giant 'spaghetti' definitions.

![pins](https://user-images.githubusercontent.com/29973601/27949358-568f3e36-62f5-11e7-845a-d928ebb20ffe.PNG)

Currently, as a workaround I use untitled Code Blocks to pin my cables in place (as above), however this might be confusing to new users and sometimes it's not immediately obvious which code blocks are performing actions and which are just 'pins'.

**Comment by Racel** [comment]
Thanks for submitting this @OliverEGreen. This is a good idea, but I am not sure when it will make it into the new UI. By you creating this issue, I was able to add it to our UX/UI backlog for future consideration. 

But, after close examination of our backlog we have a considerable amount of other higher priority tasks that we have to get done before this. Since we will likely not take this on in the next few months, I will close this for now. But, know that it is now on our radar and will move up in priority as we are able to close other tasks.

**Comment by Amoursol** [comment]
@OliverEGreen - This is now being scoped to get into Dynamo.

**Comment by mbuttery-ghd** [comment]
@Amoursol - Can this be closed?

**Comment by Amoursol** [comment]
@mbuttery-ghd indeed! Thank you for the catch. We should do another pass here and lose things.
